Nos/Views/HomeTab.swift

```diff
@@ -58,6 +35,25 @@ struct HomeTab: View {
                     }
                 }
             }
+            .navigationDestination(for: ReplyToNavigationDestination.self) { destination in
+                RepliesView(note: destination.note, showKeyboard: true)
+            }
+            .navigationDestination(for: StoriesDestination.self) { stories in
```

Comment:
Refactored this so that Stories are opened by pushing to the router path. This allows us to open stories from any part of the app, to open it with an author already selected, and it feels more switui-ty. For the end user, this loses the icon rotation though